src/main/java/net/unicon/lti/controller/lti/OIDCController.java

```diff
@@ -120,28 +125,52 @@ public String loginInitiations(HttpServletRequest req, Model model) {
                 model.addAttribute("client_id_received", clientIdValue);
                 model.addAttribute("deployment_id_received", deploymentIdValue);
             }
-            // This can be implemented in different ways, on this case, we are storing the state and nonce in
-            // the httpsession, so we can compare later if they are valid states and nonces.
+
+            
+            
+            //TODO: Here is were we need to start the process for the cookies.
+            //First, we need to know if the cookie solution is enabled for the LMS.
+            //That will happen looking at the lti_storage_target in the loginInitiationDTO. If not null, we will use
+            //the "no cookies" solution.  If null, we will need to use the old code
+            String stateHash = parameters.get("state");
```

Comment:
Could you change the name of the `parameters` variable to be `authRequestParameters` so that it's more descriptive?